tests/garage/tf/experiment/test_local_tf_runner.py

```diff
@@ -140,7 +142,8 @@ def test_make_sampler_local_sampler(self):
             assert isinstance(runner._sampler, LocalSampler)
             runner.train(n_epochs=1, batch_size=10)
 
-    def test_make_sampler_ray_sampler(self, ray_session_fixture):
+    @pytest.mark.mujoco
+    def test_make_sampler_ray_sampler(self, ray_session_fixture):  # noqa: F811
```

Comment:
can you make this test not depend on MuJoCo? it doesn't seem necessary for the test.